common/src/main/java/com/microsoft/identity/common/internal/net/ObjectMapper.java

```diff
@@ -0,0 +1,47 @@
+package com.microsoft.identity.common.internal.net;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.UnsupportedEncodingException;
+import java.lang.reflect.Type;
+import java.net.URLEncoder;
+import java.util.Iterator;
+import java.util.TreeMap;
+
+public class ObjectMapper {
+
+    public final static String ENCODING_SCHEME = "UTF-8";
+
+    public static String serializeObjectToJsonString(Object object) {
+        return new Gson().toJson(object);
+    }
+
+    public static <T> T deserializeJsonStringToObject(String json, Class<T> objectClass) {
+        return new Gson().fromJson(json, objectClass);
+    }
+
+    public static String serializeObjectToFormUrlEncoded(Object object) throws UnsupportedEncodingException {
```

Comment:
Is my memory correct in thinking there was some particular quirk of Gson that this method leveraged which we'd planned to document inline somewhere? Or was that another method?